client/src/pages/[feed]/index.tsx

```diff
@@ -44,24 +42,56 @@ const Feed: NextPage<FeedPageInitials> = (props): JSX.Element => {
 	const [titleImageBlob, setTitleImageBlob] = useState<Blob | null>(null)
 	const [titleNotFound, setTitleNotFound] = useState(props.error)
 
+	const [tagResult, setTagResult] = useState([])
+	const [noTagDataMessage, setNoTagDataMessage] = useState('Enter at least 3 characters to search')
+	const [tagValue, setTagValue] = useState(title.attributes.tags)
+
 	const [form] = Form.useForm()
 
+	useEffect(() => {
+		if (title) handleEntryFetching(10 * (Number(router.query.page) - 1) || 0)
```

Comment:
`Number(router.query.page)` could be `NaN` or negative value. 
```suggestion
		if (!title) return
		const queryPage = isNaN(+(router.query.page)) ? 1 : +(router.query.page)
		handleEntryFetching(Math.max(10 * (queryPage - 1), 0))
```